frontend/src/components/GameComponent/gameFinal.js

```diff
@@ -1,19 +1,74 @@
 import Game from './gameTop/Game';
 import Nav from './gameSectionNav/Nav';
 import Prices from './gameparts/Prices';
+
 import About from './gameparts/About';
+import { useParams } from 'react-router-dom';
+import particularData from '../HomeComponents/UpcomingGames/gameData';
+import { useState, useEffect } from 'react';
+
+function GameFinal() {
+  const {id}=useParams()
+  console.log(id)
+  const [gameData, setGameData] = useState()
+  useEffect(() => console.log(gameData), [gameData])
+
+  useEffect(() => {
+    fetch(`http://localhost:4000/gameGet/${id}`).then(res=>res.json()).then(data=>{
```

Comment:
It's better to put `http://localhost:4000/gameGet` in a env variable instead of putting links/keys data.